### DISCUSSION TITLE
Save state in Brackets

### DISCUSSION THREAD

**Discussion by ChaosCorpInc** [brackets-cont/brackets#84] [Feature Request]
I am new to Brackets and really liking what I am seeing so far. But is it possible to set something that saves what is being worked on in some form of cache, so you can shut down the program without saving text files and start it up again with everything in place to work further.

This also extends to if the program is force shut down by windows. Currently you just lose everything you've worked on that is not saves, while in something like notepad++ everything is restored.

**Comment by abose** [comment]
Thanks @ChaosCorpInc Would be a good feature to have. 
Moved to Feature requests discussions.
